ls1intum/ArTEMiS
# Added confirmation dialog for submitting assessment

### Checklist

### Motivation and Context


This should resolve the issue #355.

### Description

I added a window submit, if user didn't assess all UML elements. If he then decides to close it, not assessed elements are highlighted. 

### Steps for Testing


1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Select Tutor Course Dashboard
4. Select Exercise Dashboard for any modelling exercise
5. Select Open Assessment
6. Click Submit with no or part of the elements assessed to see confirmation dialog

### Screenshots


![Partially asssessed](https://user-images.githubusercontent.com/17262376/58754509-07c72100-84d2-11e9-9b25-3893850b0e3e.png)
![Warning dialog](https://user-images.githubusercontent.com/17262376/58754513-17466a00-84d2-11e9-9114-f326bb343eaa.PNG)
![Unassesed elements highlighted](https://user-images.githubusercontent.com/17262376/58754511-144b7980-84d2-11e9-8ff7-a19afab0a98b.png)
